src/ScriptRunner/ScriptRunner.GUI/ViewModels/MainWindowViewModel.cs

```diff
@@ -476,6 +476,8 @@ public async void PasteParametersSetup()
             try
             {
                 var actualPayload = payload.Substring(first, (last+1) - first);
+                //INFO: When you send JSON via MSTeams, you will get extra non-breaking spaces on the other side. Those extra white-spaces breaks deserializer
+                actualPayload = actualPayload.Replace(" \u00A0", " ");
```

Comment:
Wouldn't it be enough to do:

```
actualPayload = actualPayload.Replace("\u00A0", "");
```

Reply:
I thought about that, but what if the value contains intentionally non-breaking space? For me, it's a valid use case. On the other hand, non-breaking space following regular space seems to be a little weird.

Reply:
that's fine with me